### PR TITLE
Sitemaps: improve comprehension

### PR DESCRIPTION
**Pull request by bgilmer77**
- made changes to improve comprehension in introduction
- will make additional commits as I work through the entire article

Signed-off-by: Brad Gilmer <brad@gilmer.tv> (github: bgilmer77)

### CODE REVIEW

File: configuration/sitemaps.md [hunk 1]
```diff
@@ -270,11 +274,13 @@ Selection item=LR_TV_Channel label="TV Channel" mappings=[0="DasErste", 1="BBC O
 Setpoint item=<itemname> [label="<labelname>"] [icon="<iconname>"] minValue=<min value> maxValue=<max value> step=<step value>
 ```
 
-A special Switch-like element to increase or decrease the value of an item.
+A special Switch-like element that may be used to increase or decrease the value of an item.
 The element is often used to gradually change a number item.
 
+<!-- TODO: The sentence above is not clear and needs to be rewritten. -->
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: I'm not even sure if a setpoint or slider can be used with anything other than a Number Item. Would you please do a quick test?

If nothing surprising comes up you can combine both sentences and state "of a Number Item" for both elements.

File: configuration/sitemaps.md [hunk 2]
```diff
@@ -354,12 +360,12 @@ Image [item=<itemname>] [icon="<iconname>"] url="<url of image>" [label="<labeln
 ```
 
 This element type is able to present an image.
-The image has to be available on a reachable website or webserver without password or access token.
-It's also possible to place an image in the `html` folder under your configuration folder.
+The image must be available on a reachable website or webserver without password or access token.
+It's also possible to place an image in the `$OPENHAB_CONF/html` folder.
 The file will be available under the "static" route, [http://<my.openHAB.device>:8080/static/image.png](http://127.0.0.1:8080/static).
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: "Such a file" ?

File: configuration/sitemaps.md [hunk 3]
```diff
@@ -376,14 +382,14 @@ Image url="https://raw.githubusercontent.com/wiki/openhab/openhab/images/feature
...
+Note: not all video encodings (formats) are supported; you may need to transcode your video.
+The video must be reachable directly via URL.
+An embedded and/or protected video are not supported.
 
-- `item` can refer to a String Item whose state is an URL to a video.    Some clients may not (yet) consider `item`.
-- `url` is the default URL from which to retrieve the video, if there is no associated Item or if the associated item's state is not an URL.
-- `encoding` can stay left empty for auto selection, for an MJPEG video please set the "mjpeg" encoding explicitly.
+- `item` can refer to a String Item whose state is a URL to a video. Some clients may not (yet) consider `item`.
+- `url` is the default URL from which to retrieve the video if there is no associated Item or if the associated item's state is not a URL.
+- `encoding` may be left empty for auto selection.  but please set it explicitly to "mjpeg" for an MJPEG video, or to "HLS" for an HTTP Live Streaming playlist (file with .m3u8 extension).
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: "But"

I'd actually turn it around.

> "mjepg" for ..., "HLS" for ..., empty otherwise.

File: configuration/sitemaps.md [hunk 4]
```diff
@@ -451,15 +459,17 @@ Splits a String Item at each separator into multiple rows.
 
 Mappings is an optional parameter for the [Switch](#element-type-switch) and [Selection](#element-type-selection) element types.
 
-Please be aware of the fact, that both Switch and Selection are input element types.
-If you are looking to transform Item data into meaningful outputs, a [Text element](#element-type-text) with it's label parameter may be a better choice.
+Please note that Switch and Selection are input-only element types.
+In other words, they only accept commands.
+For more information, see [Commands vs. Status]({{base}}/configuration/items.md#command-vs-status) in the [Items]({{base}}/configuration/items.md) article.
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: I might be the author of the original version of this. I have to admit that it is misleading.

A Switch or a Selection can of course also be used to represent the current state. I wanted to highlight, that a text element is recommended IF the underlying Item can not be written (e.g. Time of Day). Not sure how this relates to Commands vs Status.  Maybe remove this paragraph all together?

File: configuration/sitemaps.md [hunk 5]
```diff
@@ -451,15 +459,17 @@ Splits a String Item at each separator into multiple rows.
...
+
+Mapping syntax:
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: I tried to reduce these short paragraphs as much as possible as they do not look good in the final document. That's why there was a bigger paragraph in between the two code blocks here.

File: configuration/sitemaps.md [hunk 6]
```diff
@@ -451,15 +459,17 @@ Splits a String Item at each separator into multiple rows.
...
 
 ```perl
 mappings=[value_1="description_1", value_2="description_2", ...]
 ```
 
-Above you can see the general syntax for the mappings parameter.
-Let's have a look at a few examples:
+Examples:
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: Should be bold, just as the rest of the "Example:" lines

File: configuration/sitemaps.md [hunk 7]
```diff
@@ -471,33 +481,46 @@ mappings=[15="Gone", 19="Chilly", 21="Cozy"]
...
 As you can see, different Item data types are accepted as mappings values.
 The first two lines show very typical use cases.
-Imagine your TV as part of your openHAB setup.
-It's power state and channel number are internally represented by a binary Switch Item (OFF/ON) and a discrete number Item with only a few selectable states.
-By using a Switch or Selection element with a mappings array, you can replace these meaningless values by meaningful descriptions for your inputs in the user interface.
+Imagine your TV is part of your openHAB setup.
+Its power state is represented by a binary Switch Item.
+Its channel number is a discrete number Item that may only be set to one of three states.
+By using a Switch or Selection element with a mappings array, you can replace these meaningless values with user-friendly descriptions for display on the user interface.
+Using this mapping, the power state of the TV, instead of saying "ON" and "OFF" would more accurately display "on" and "standby".
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: Add new line here?

"this mapping"?

File: configuration/sitemaps.md [hunk 8]
```diff
@@ -471,33 +481,46 @@ mappings=[15="Gone", 19="Chilly", 21="Cozy"]
...
+- Present a special icon, depending on the state of an item (a [dynamic icon]({{base/items.md#icons-dynamic}}))
 
 ### Visibility
 
-To dynamically show or hide an item, the `visibility` parameter is used.
-By default, an Item is visible if the `visibility` parameter is not provided.
+The `visibility` parameter is used to dynamically show or hide an Item.
+If the parameter is not provided, the default is to display the Item.
+
+Visibility syntax:
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: See above. I believe these are superfluous?

File: configuration/sitemaps.md [hunk 9]
```diff
@@ -471,33 +481,46 @@ mappings=[15="Gone", 19="Chilly", 21="Cozy"]
...
 ```
 
-The format of the `visibility` parameter is given above, let's also look at a few examples:
+Valid comparison operators are:
+
+- equal to `==`, unequal to `!=`
+- smaller or equal to `<=`, bigger or equal to`>=`
+- smaller than `<`, bigger than `>`
+
+Expressions are evaluated from left to right.
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: Does this one has to be it's own paragraph?

File: configuration/sitemaps.md [hunk 10]
```diff
@@ -471,33 +481,46 @@ mappings=[15="Gone", 19="Chilly", 21="Cozy"]
...
+
+Examples:
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: See above.

File: configuration/sitemaps.md [hunk 11]
```diff
@@ -506,29 +529,33 @@ visibility=[Day_Time=="Morning", Day_Time=="Afternoon", Temperature>19]
...
 If any one of the comparisons is evaluated as `true`, the Item will be visible, otherwise it will be hidden.
-It is important to note, that it is not possible to decide visibility on more than one condition at the same time.
-The third example might be the visibility of a sprinkler control.
-The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
-To achieve more complex conditions, you will benefit from defining a helper Item and a rule to set it.
+It is important to note that visibility may only be determined based on one condition at a time.
 
-Valid comparison operators are:
+The third example might control the visibility of a lawn sprinkler control.
+The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: ... a combination of multiple conditions is not supported...

File: configuration/sitemaps.md [hunk 12]
```diff
@@ -506,29 +529,33 @@ visibility=[Day_Time=="Morning", Day_Time=="Afternoon", Temperature>19]
...
+To achieve more complex conditions, you will benefit from defining a helper Item and a [rule]({{base}}/configuration/rules-dsl.md) to set it.
 
-- equal to `==`, unequal to `!=`
-- smaller or equal to `<=`, bigger or equal to`>=`.
-- smaller than `<`, bigger than `>`
+<!-- TODO: Provide an example for the statement above or remove it.  It is not that helpful as it is now for people who are just getting aquainted with sitemaps -->
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: Which?

File: configuration/sitemaps.md [hunk 13]
```diff
@@ -506,29 +529,33 @@ visibility=[Day_Time=="Morning", Day_Time=="Afternoon", Temperature>19]
...
 
 ### Label and Value Colors
 
 Colors can be used to emphasize an items label or its value based on conditions.
+Colors may be assigned to either the label or the value associated with an Item.
+
+Label and Value Color Syntax:
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: See above

File: configuration/sitemaps.md [hunk 14]
```diff
@@ -506,29 +529,33 @@ visibility=[Day_Time=="Morning", Day_Time=="Afternoon", Temperature>19]
...
-The general format of the `labelcolor` and `valuecolor` parameters is given above.
-The comparison operators are the same as for the mappings parameter.
-Let's have a look at a few examples:
+Note that `item_name` and `operator` are both optional.
+If `item_name` is not provided, the Item name will default to the current Item.
+If an operator is not specified, the operator will default to `==`.
+
+The comparison operators for `labelcolor` and `valuecolor` are the same as for the visibility parameter.
+
+Examples:
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: See above

File: configuration/sitemaps.md [hunk 15]
```diff
@@ -537,16 +564,16 @@ Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
...
-In the first example, the `Weather` Item is colored blue (label and value) if the `Temperature` is below or equal to 4°C.
+In the first example, the `Weather` Item is blue (label and value) if the `Temperature` is at or below 4°C.
 
-Expressions will be evaluated from left to right.
-The first condition returning `true` will decide on the color.
-Looking at the second example, you will see, that the given five expressions are ordered in the only meaningful combination.
+The order in which expressions are listed is important.
+Note that expressions are evaluated from left to right, and the first matching expression determines the color.
+Looking at the second example, you will see that the order of the expressions is set so that the first match, left-to-right gives the correct result.
+If the order was reversed, color assignment would not work properly.
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: Not sure if this is clearer than the original.

File: configuration/sitemaps.md [hunk 16]
```diff
@@ -537,16 +564,16 @@ Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
...
 
-`item_name` and `operator` are both optional.
-If not provided, the Item name will default to the current Item and operator will default to `==`.
-In the second example, this is shown by leaving out Temperature and by not giving a comparison operator in the expression `0="white"`.
-The following three lines are totally equal and valid:
+In the second example, the effect of omitting the `item_name` and `operator` parameters is shown by omitting `Temperature` and by omitting the comparison operator in the expression `0="white"` (as compared to `==0="white"`).
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: Maybe not needed?

File: configuration/sitemaps.md [hunk 17]
```diff
@@ -537,16 +564,16 @@ Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
...
+
+The following three lines are equivalent, and all of them are valid:
```
Comment (ThomDietrich) [inline configuration/sitemaps.md]: I wonder if these should be the first example?

File: configuration/sitemaps.md [hunk 18]
```diff
@@ -130,23 +130,27 @@ The following element types can be used in a Sitemap definition file.
 Data presented by Sitemap elements will almost always originate from a referenced item.
```
[the remark this answers is not in the document]
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Please correct all occurrences of "item"

File: configuration/sitemaps.md [hunk 19]
```diff
@@ -90,38 +90,38 @@ Xtext Domain Specific Language and the sitemap file model can be found [here](ht
...
 | [Group](#element-type-group)              | Concentrates all elements of a given group in a nested block. |
 | [Image](#element-type-image)              | Renders an image given by an URL. |
 | [Selection](#element-type-selection)      | Provides a dropdown or modal popup presenting values to choose from for an Item. |
 | [Setpoint](#element-type-setpoint)        | Renders a value between an increase and a decrease buttons. |
-| [Slider](#element-type-slider)            | A value is presented in a progress bar like slider. |
+| [Slider](#element-type-slider)            | Presents a value in a progress-bar-like slider. |
 | [Switch](#element-type-switch)            | Renders an Item as a ON/OFF or multi-button switch. |
-| [Text](#element-type-text)                | Renders an Item in a text representation. |
-| [Video](#element-type-video)              | Displays a video stream given a direct URL. |
+| [Text](#element-type-text)                | Renders an Item in as text. |
```
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: ??  So this sentence reads, "[Text] renders an Item in as text.", if I understand your correction.  Also you dropped the video element?  Is that feature deprecated?

Rendering an Item in as text makes no sense to me.  Rendering an Item as text makes sense.

File: configuration/sitemaps.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: Okay - will change all occurences of "item" to "Item".

File: configuration/sitemaps.md [hunk 20]
```diff
@@ -130,23 +130,27 @@ The following element types can be used in a Sitemap definition file.
 Data presented by Sitemap elements will almost always originate from a referenced item.
 Each Item is of a certain Item type, for example `Switch`, `Number` or `String`.
 
-While not all combinations are meaningful, items of one datatype can be linked to different Sitemap element types.
+While not all combinations are meaningful, items of one datatype may be linked to different Sitemap element types.
 This provides the flexibility to present items in the way desired in your home automation user interface.
 
+<!-- TODO: an example of the above would be useful here -->
+
```
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: Okay - I will look at this.

File: configuration/sitemaps.md [hunk 21]
```diff
@@ -130,23 +130,27 @@ The following element types can be used in a Sitemap definition file.
...
 
 -   Common parameters, also known from [items definition](items.html#item-syntax):
     - `item` defines the name of the Item you want to present (e.g. `Temperature`), [more details](items.html#item-name).
-    - `label` sets the textual description besides the preprocessed Item data (e.g. "`Now [%s °C]`"), [more details](items.html#item-label).
-    - `icon` is the name of the icon file to show next to the element, [more details](items.html#icons).
+    - `label` sets the textual description displayed next to the preprocessed Item data (e.g. "`Now [%s °C]`"), [more details](items.html#item-label).
+    - `icon` chooses the name of the icon file to show next to the element, [more details](items.html#icons).
 
--   Setting a value for `label` or `icon` of a Sitemap element will override the values defined for the linked Item.
+-   When an Item is defined, you have the opportunity to assign a label and/or an icon at that point.
```
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: Hum - learning here.  What is a Link Item?  Is it a defined thing?

File: configuration/sitemaps.md [hunk 22]
```diff
@@ -19,13 +19,13 @@ This page is structured as follows:
...
-A Sitemap definition file is stored in the folder `$OPENHAB_CONF/sitemaps` and has to have the `.sitemap` filename extension.
-For easy editing, profit from the IDE support of the [openHAB supporting editors]({{base}}/configuration/editors.html).
+Sitemaps are text files with the `.sitemap` extension, and are stored in the `$OPENHAB_CONF/sitemaps` directory.
+Sitemaps follow the syntax described in this article.
 
-The openHAB runtime comes with a demo configuration package containing a [`demo.sitemap`](https://github.com/openhab/openhab-distro/blob/master/features/distro-resources/src/main/resources/sitemaps/demo.sitemap),
-which should let you easily understand possible elements and structures.
-It is recommended to use this demo file or another example Sitemap as a starting point towards building a customized Sitemap that fits your personal home setup.
+For easy editing of sitemap definition files, we suggest you use [Eclipse SmartHome Designer]({{base}}/configuration/editors.html#esh-designer), which provides full IDE support for these files including, among other functions, syntax checking and auto-complete.
```
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: So how about something like this...

"We suggest you use an editor that includes support for openHAB syntax such as Atom or Eclipse SmartHome Designer.  These editors provide syntax checking, auto-complete capability and more."

File: configuration/sitemaps.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: Will change to "Renders an Item as text."

File: configuration/sitemaps.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: What I meant is that a little explanation would be helpful.  How about adding a sentence just above the TODO as follows:
"For example, a Temperature Item may be rendered as text to display the temperature in degrees C.  It may also be rendered as a Slider Element, allowing a user to set the temperature in a room to a specific setting."

File: configuration/sitemaps.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Sounds good!

File: configuration/sitemaps.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (ThomDietrich) [reply configuration/sitemaps.md]: 🙂👍

File: configuration/sitemaps.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Ups. I meant: Please add a link to the Items article here

**Comment by ThomDietrich** [comment]
@bgilmer77 I'll merge this one soon when I have a few more minutes. I think it is finished.
Just to make you aware: It seems that you've deleted the repository or branch it was based on. For the PR GitHub still provides the changes but normally they would be gone now. Whatever you did... you might not want to do that again 😄

**Comment by bgilmer77** [comment]
Umm. Yes - Mea Culpa.  Honestly I don't remember what happened.  I have managed to get myself into various bad places with Git as I have been trying to get the hang of things.

BTW, it looks to me as if you have made a number of comments that I have not yet addressed.  Do you know if this is true?  Yuck!  I see now that you are correct - I no longer have that branch on my repo.  Hum.

How about this - go ahead and merge.  Then I will look at your comments here and if they are still unaddressed, I will create a new branch, address your comments, and create a new PR.  Is that okay, or is there a better way to tackle this?

**Comment by ThomDietrich** [comment]
Hey brad, fine with me. Let's do that. Addressing these comments would be important to me as I remember that some of them were about changes that imho were not correct or complete.

Btw. Some of your commits were not signed-off. Please remember to always do so. Many git clients offer the option to do that automatically.

File: configuration/sitemaps.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: Will do.

File: configuration/sitemaps.md (hunk 2 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Um... how about replacing from "It's also possible..." with,

"Alternatively, the image may be stored locally in the `$OPENHAB_CONF/html` folder, and will be accessible through the static route, [http://<my.openHAB.device>:8080/static/image.png](http://127.0.0.1:8080/static)."

File: configuration/sitemaps.md (hunk 3 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This is what I put in my commit:

- `encoding` should be set to "mjpeg" for an MJPEG video, or "HLS" for an HTTP Live Streaming playlist (file with .m3u8 extension).  If you omit the `encoding` parameter, openHAB will attempt to automatically select the correct format.

File: configuration/sitemaps.md (hunk 4 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: I agree that this paragraph should be deleted.

File: configuration/sitemaps.md (hunk 5 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: So I deleted this paragraph per the above.

File: configuration/sitemaps.md (hunk 6 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Agree

File: configuration/sitemaps.md (hunk 7 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Adding a new line and rewording as below:
This mapping changes the displayed power state of the TV from "ON" and "OFF" to the more accurate terms, "on" and "standby".
Similarly, mapping above changes the numbers "1", "2", and "3" to "DasErste", "BBC One", and "Cartoon Network" respectively.

File: configuration/sitemaps.md (hunk 8 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Hum - not sure how this is superflurious - or even not needed ;-)

File: configuration/sitemaps.md (hunk 9 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Not at all - removed extra line above and below the bullet points.

File: configuration/sitemaps.md (hunk 10 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Agreed

File: configuration/sitemaps.md (hunk 11 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Dropping the "Third example".

File: configuration/sitemaps.md (hunk 12 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Now it is my turn to have confused you with a short comment ;-).

So the line above starts out, "To achieve more complex conditions, you will benefit from defining a helper Item..."

What is the meaning of "To achieve more complex conditions"?
What is a "helper Item"?

These seem like more advanced concepts and can be confusing as it is currently written.

I think I understand the first point - maybe it is trying to say, "If you are trying to do something more complicated than these simple examples, then you could define a helper Item..."

But I still don't know what a helper Item is.  A short simple example would really be good - otherwise I suggest we delete the whole idea (but that would be too bad, since then people would have to dig other places to find out how to do more complex things (but hey - they have to do that anyway for a lot of other stuff (Oops - nested parentheses again.))).

File: configuration/sitemaps.md (hunk 13 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: I do not understand this comment.

File: configuration/sitemaps.md (hunk 14 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Yep - I assume you mean to have Examples be in bold.

Comment (bgilmer77) [reply configuration/sitemaps.md]: Yep - bold "Examples:"

File: configuration/sitemaps.md (hunk 15 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Hum - don't know what to say.  And now I do not have the original?

File: configuration/sitemaps.md (hunk 16 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Oh - I don't know...  I have not worked with colors in sitemaps before, and I found this second example helpful.  Up to you.  Let me know if you want to kill it.

File: configuration/sitemaps.md (hunk 17 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Actually, I think this group of examples needs to be redone.  Here is what I would propose:

**Examples**
The following three lines are equivalent:

```perl
Text item=Temperature labelcolor=[>0="blue"] valuecolor=[22="green"]
Text item=Temperature labelcolor=[>0="blue"] valuecolor=[==22="green"]
Text item=Temperature labelcolor=[>0="blue"] valuecolor=[Temperature==22="green"]
```

The line below illustrates the importance of operator order:
```perl
Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
                                  >=25="orange", >=15="green", 0="white", <15="blue"]
```
Note that expressions are evaluated from left to right, and the first matching expression determines the color.
Looking at the line above, you will see that the first match, left-to-right gives the correct result.
If the order was reversed, the color assignment would not work properly.

Note also, the effect of omitting `Temperature` and omitting the comparison operator in the expression `0="white"` (as compared to `==0="white"`).

Comment (ThomDietrich) [reply configuration/sitemaps.md]: Sounds good.

File: configuration/sitemaps.md (hunk 16 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Let's leave it in, your rewording in the other comment should do the trick

File: configuration/sitemaps.md (hunk 15 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: You have, you can always check the git history of a file...

File: configuration/sitemaps.md (hunk 14 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Yes

File: configuration/sitemaps.md (hunk 13 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Bold?

File: configuration/sitemaps.md (hunk 12 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Here's what the sentence intends to say: "If you are trying to do something more complicated than these simple examples, then you could define an additional Item which is set by a rule based on your more complex idea"

Upon writing this I agree to leave the whole sentence out.

File: configuration/sitemaps.md (hunk 11 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Not sure if we are on the same page. The example is important, I just think that it would be clearer if we specifically stated, that OR means that combinations are indeed not supported

File: configuration/sitemaps.md (hunk 8 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Basically I don't like these two-word paragraphs. They tend to look not good in the end result. However yes sure, without them the read flow is interrupted by a sudden code block. Do as you see fit

File: configuration/sitemaps.md (hunk 2 above)
Comment (ThomDietrich) [reply configuration/sitemaps.md]: Sounds good.
In other articles we used  http://openhab-device:8080 and upon reading the sentence, it feels like image.png comes a bit out of nothing... not sure how to make clear that the name is an example.

File: configuration/sitemaps.md (hunk 1 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: I have a development system here, so no problem to do a test.  But what do you suggest?  Is your point to test something like a Switch Item to see if the Setpoint can be used to control it?

File: configuration/sitemaps.md (hunk 2 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Here is what I will put in my next commit:

Alternatively, the image file (e.g. YourImageFile.png) may be stored locally in the $OPENHAB_CONF/html folder, and will be accessible through the static route, http://<my.openHAB.device>:8080/static/YourImageFile.png.

File: configuration/sitemaps.md (hunk 11 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Yes, we are not on the same page.

So I dropped the example that is referred to by: "The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C."

I thought you were saying that this example with the OR logic is broken and therefore should be removed.  In your note above, I am failing at parsing your sentence, "it would be clearer if we specifically stated, that OR means that combinations are indeed not supported".  What do you mean by this?

File: configuration/sitemaps.md (hunk 12 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Actually, I quite like the idea of leaving "bread crumbs" for people who may be trying to do something more than just the basics.  Many times I find that all I need is a small hint about how to attack a more complex issue and I can take it from there.

How about we change the "To achieve more complex..." sentence to:

If you are trying to do something more complicated than these simple examples, then you could define an additional `Item` which is set by a `Rule`.
Rules have a rich set of features that accommodate more complex scenarios.

File: configuration/sitemaps.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 3 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 4 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 5 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 6 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 7 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 9 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 10 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed

File: configuration/sitemaps.md (hunk 11 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: IMPORTANT - is this sentence, which is currently submitted as a change, correct?

"It is important to note that visibility may only be determined based on one condition at a time."

FWIW, if it is correct, I would reword this to, "It is important to note that visibility is determined based upon only one condition at a time."

File: configuration/sitemaps.md (hunk 13 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: Understood - this comment has been addressed.

File: configuration/sitemaps.md (hunk 14 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 16 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 17 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 2 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 8 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: This comment has been addressed.

File: configuration/sitemaps.md (hunk 15 above)
Comment (bgilmer77) [reply configuration/sitemaps.md]: For now, let's just go with this wording and move on.  I will figure out Git History at some other point.
This comment has been addressed.
